### ISSUE TITLE
False positive in plugin-next / no-page-custom-font

### ISSUE THREAD

**Issue by steelbrain** [oxc-project/oxc#3378]
Hello,

I am running oxlint v0.3.5 and it's reporting a false positive for `no-page-custom-font` rule for the following code

```typescript
<link
  rel="apple-touch-startup-image"
  href="/assets/public/pwa/splash/apple-splash-2048-2732.jpg"
  media="(device-width: 1024px) and (device-height: 1366px) and (-webkit-device-pixel-ratio: 2) and (orientation: portrait)"
/>
```

As you can see, this is a splash image `link` element. This is not related to custom fonts at all. Please let me know if there's additional information I can provide here

**Comment by Dunqing** [comment]
Thank you for your feedback! You have provided enough information.

**Comment by steelbrain** [comment]
Thank you for resolving this so quickly! ⚡
